python/itools-y4m.py

```diff
@@ -185,10 +175,6 @@ def read_frame(self, data, output_colorrange, debug):
         # 2. read luminance
```

Comment:
please fix the comments. Now you're not reading the luminance here, just calculating the frame values